app/config/tstvhq/rooms.json

```diff
@@ -165,7 +165,7 @@
             "name": "Barnyard",
             "type": "IFRAME",
             "iframeOptions": {
-                "src": "https://player.twitch.tv/?channel=disasterina&parent=estate.tstvhq.com"
+                "src": "https://youtu.be/N_DwST8vKdQ"
```

Comment:
I don't think shortened urls are going to work

Reply:
ah! thanks. its cool, we can skip this-- was just placeholder